### DISCUSSION TITLE
How to freely rotate map by hand and come back to north?

### DISCUSSION THREAD

**Discussion by Piotr-Wosinek** [liodali/osm_flutter#406] [Q&A] [answered]
There is change orientation of the map (await controller.rotateMapCamera(degree);) but this is something else.
Map is static and directed to north but I want to freely rotate map by hand like in Google map.

I thought about InteractiveViewer() before OSMFlutter widget but I don't know what is best?

Piotr

**Comment by liodali** [comment]
you mean when rotate the phone the map also rotate or by gesture ?

**Reply by Piotr-Wosinek** [reply]
by gesture

**Reply by liodali** [reply, marked as answer]
i will add attribute to active or disactive that behavior for now is disactivated by default

**Reply by Piotr-Wosinek** [reply]
Thank you, I hope to see this soon ;)
